diagnostic.md

```diff
@@ -49,10 +56,22 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+    I am not entirely sure what you mean by 'value'.
+    here is how you would get to that route in the router
+
+    this.route('movie', { path: '/movies/:movie_id'});
+
+    and in the template you could reference the value 123 like this.
+    {{#link-to 'movie' movie}}{{movie._id.123}}{{/link-to}}</li>
+
+
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
     <!-- your response here -->
+    you reference sort of like how in mongo. like object.attribute. So say you have
+    a house route that has attribute diningRoom it'd be referenced like
+    {{house.diningRoom}} in handlebars moustaches.
```

Comment:
Yes, you might be able to access it that way inside a component template. In a route template, we generally use `model`.
.5/1